hail/python/hail/context.py

```diff
@@ -552,10 +554,12 @@ def init_local(
     tmpdir = _get_tmpdir(tmpdir)
     optimizer_iterations = get_env_or_default(_optimizer_iterations, 'HAIL_OPTIMIZER_ITERATIONS', 3)
 
+    jvm_heap_size = get_env_or_default(jvm_heap_size, 'HAIL_LOCAL_BACKEND_HEAP_SIZE', None)
```

Comment:
Can you add an `export HAIL_LOCAL_BACKEND_HEAP_SIZE=3750M` to build.yaml for local tests?